CHANGELOG.md

```diff
@@ -6,7 +6,12 @@ This gem is moving onto its own [Semantic Versioning](https://semver.org/) schem
 
 Prior to v1.0.0 this gem was versioned based on the `MAJOR`.`MINOR` version of RuboCop. The first release of the ezcater_rubocop gem was `v0.49.0`.
 
+## v4.0.0
+
+- Update to Ruby 3.1 and latest dependency gems
```

Comment:
Is this a breaking change, as implied by the major version bump? If so, is there a way that we could support ruby 2.7 and 3.1 simultaneously?

Reply:
No, I must have misunderstood what was being asked. Let me try a new PR here.